src/Microsoft.ML.StandardLearners/Standard/MultiClass/Pkpd.cs

```diff
@@ -26,7 +26,7 @@ namespace Microsoft.ML.Runtime.Learners
 {
 
     using TDistPredictor = IDistPredictorProducing<float, float>;
-    using TScalarTrainer = ITrainerEstimator<IPredictionTransformer<IPredictorProducing<float>>, IPredictorProducing<float>>;
+    using TScalarTrainer = ITrainerEstimator<IClassicPredictionTransformer<IPredictorProducing<float>>, IPredictorProducing<float>>;
```

Comment:
>Classic [](start = 46, length = 7)

Yes after seeing this a bit I think the word "classic" is just going to confuse the heck out of a lot of people. Please consider doing something else.